core/deliverservice/config.go

```diff
@@ -124,6 +130,24 @@ func (c *DeliverServiceConfig) loadDeliverServiceConfig() {
 	}
 	c.BlockGossipEnabled = enabledConfigOptionMissing || viper.GetBool(enabledKey)
 
+	blockCensorshipTimeout := "peer.deliveryclient.blockCensorshipTimeout"
+	blockCensorshipTimeoutOptionMissing := !viper.IsSet(blockCensorshipTimeout)
+	if blockCensorshipTimeoutOptionMissing {
+		c.BlockCensorshipTimeout = DefaultBlockCensorshipTimeout
+		logger.Infof("peer.deliveryclient.blockCensorshipTimeout is not set, defaulting to 30s.")
```

Comment:
you have

```
   viper.SetDefault(blockCensorshipTimeout, DefaultBlockCensorshipTimeout)
```

and there is no need to check whether the value was set or not.

Reply:
So you suggest replacing the code (lines 133-140) by the following code:

	blockCensorshipTimeout := "peer.deliveryclient.blockCensorshipTimeout"
	c.BlockCensorshipTimeout = viper.GetDuration(blockCensorshipTimeout)
	viper.SetDefault(blockCensorshipTimeout, DefaultBlockCensorshipTimeout)